samples/outbox/sqltransport-sqlpersistence-ef/SqlServer_3/Receiver/ReceiverDataContext.cs

```diff
@@ -1,30 +1,34 @@
-using System.Data.Entity;
-using System.Data;
 using System.Data.Common;
+using Microsoft.EntityFrameworkCore;
 
 public class ReceiverDataContext :
     DbContext
 {
     #region EntityFramework
-    public ReceiverDataContext(string connection)
-        : base(connection)
+    public ReceiverDataContext(DbConnection connection)
     {
+        this.connection = connection;
     }
-    public ReceiverDataContext(IDbConnection connection)
-        : base((DbConnection) connection, false)
+    public ReceiverDataContext(string connectionString)
     {
+        this.connectionString = connectionString;
     }
-    #endregion
-
-    public DbSet<Order> Orders { get; set; }
 
-    protected override void OnModelCreating(DbModelBuilder modelBuilder)
+    protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
     {
-        base.OnModelCreating(modelBuilder);
-        
-        var orders = modelBuilder.Entity<Order>();
-        orders.ToTable("Orders");
-        orders.HasKey(x => x.OrderId);
-        orders.Property(x => x.Value);
+        if (!string.IsNullOrEmpty(connectionString))
```

Comment:
The idea was to first check if x is valid before using x but your comment makes me think this is not very readable. I'll try to improve so that intent is clear.